content/blog/2020-12-10-masks-public.Rmd

```diff
@@ -0,0 +1,363 @@
+---
+title: "Are Masks Widely Used in Public?"
+author: "Alex Reinhart"
+date: 2020-12-07
+tags: ["R", "symptom surveys", "COVIDcast"]
+draft: true
```

Comment:
I can, but perhaps you want to change the template Rmd to not include `draft`?

Reply:
I get your point. However, this template post is one that is in the main branch but actually shouldn't be published, thus draft :-D. We could this template post, since we have now enough other examples to start with. What do you think?

Reply:
Fair enough. I think the alternative is to modify the template post to give instructions -- for example, to include some examples of using `blogdown::apiref`, use the better tag syntax, show how to specify the post summary, etc. And the instructions can say "remove `draft: true`".